DuckDB.NET.Data/Internal/Reader/DecimalVectorDataReader.cs

```diff
@@ -61,4 +63,14 @@ private decimal GetDecimal(ulong offset)
             default: throw new DuckDBException($"Invalid type {DuckDBType} ({(int)DuckDBType}) for column {ColumnName}");
         }
     }
+
+    internal byte GetScale() 
```

Comment:
I think it's better to turn these fields into readonly properties.

Reply:
Change it.